File: Source/WickedSick.Server.XamlParser/Elements/Media/Effects/BlurEffect.cs

namespace WickedSick.Server.XamlParser.Elements.Media.Effects
{
    [Element(NullstoneNamespace = "Fayde.Media.Effects")]
    public class BlurEffect : Effect
    {
        public static readonly PropertyDescription RadiusProperty = PropertyDescription.Register("Radius", typeof(double), typeof(BlurEffect));
    }
}

File: src/Media/Effects/BlurEffect.ts
/// <reference path="Effect.ts" />

module Fayde.Media.Effects {
    export class BlurEffect extends Effect {
        static RadiusProperty: DependencyProperty = DependencyProperty.Register("Radius", () => Number, BlurEffect, undefined, (d, args) => (<Media.Effects.Effect>d)._EffectChanged(args));
        Radius: number;
    }
    Fayde.RegisterType(BlurEffect, "Fayde.Media.Effects", Fayde.XMLNS);
}